src/mscorlib/shared/System/ArgumentException.cs

```diff
@@ -21,7 +21,7 @@ namespace System
     // message describing what was wrong and which parameter is incorrect.
     // 
     [Serializable]
-    public class ArgumentException : SystemException, ISerializable
```

Comment:
Yes - it is not necessary. It was removed in CoreRT, and it is consistent with most other CoreLib exception types.